src/layers/conv.jl

```diff
@@ -1546,7 +1551,7 @@ function (l::SGConv)(g::AbstractGNNGraph, x,
     end
     c = 1 ./ sqrt.(d)
     for iter in 1:(l.k)
-        x = x .* c'
+        !(g isa GNNHeteroGraph) ? xj = xj .* c' : Nothing
```

Comment:
this stuff is not correct, it's not like since it is not a heterograph we can skip normalization. The same goes for gcnconv. 
I think that the right thing would be to compute two normalization constants
```
cin = degree(g, dir=:in)
cout =  degree(g, dir=:out)
```
and use `cout` for the first normalization and `cin` for the second.

Reply:
I was wondering about it, something didn't feel right here. thanks, I'll amend GCN.